clients/fides-js/src/components/Button.tsx

```diff
@@ -5,17 +5,19 @@ interface ButtonProps {
   buttonType: ButtonType;
   label?: string;
   onClick?: () => void;
+  className?: string;
 }
 
 const Button: FunctionComponent<ButtonProps> = ({
   buttonType,
   label,
   onClick,
+  className = "",
 }) => (
   <button
     type="button"
     id={`fides-banner-button-${buttonType.valueOf()}`}
```

Comment:
Unrelated, but this is still a bug here - we can have multiple types of the same button in the DOM which is invalid HTML 👍

Reply:
okay—I guess we'll have to come up with a plan to fix these ids while also migrating customers who are already using these ids then.